src/methods.js

```diff
@@ -131,6 +107,9 @@ export default {
     getTransaction: { version: '>=2.2.0' },
     getUnconfirmedBalance: { version: '>=2.2.0' },
     getWalletInfo: { version: '>=2.2.0' },
+    getblocktemplate: { version: '>=3.0.0' },
```

Comment:
I noticed all of the new 3.0 methods are no longer following camelcase convention.  Can we change this so that they are?  i.e. importelectrumwallet -> importElectrumWallet

Reply:
you are right.  I made a mistake here.  nice catch!  Updated.  Please take a look again whenever you get a chance